crates/sui-indexer/src/db.rs

```diff
@@ -228,19 +239,22 @@ pub mod setup_postgres {
                 EXECUTE 'DROP FUNCTION IF EXISTS ' || quote_ident(r.proname) || '(' || r.argtypes || ') CASCADE';
             END LOOP;
         END $$;";
-        diesel::sql_query(drop_all_functions).execute(conn)?;
+        diesel::sql_query(drop_all_functions)
+            .execute(&mut conn)
+            .await?;
         info!("Dropped all functions.");
 
-        conn.setup()?;
-        info!("Created __diesel_schema_migrations table.");
+        // conn.setup()?;
```

Comment:
Is this intentional? the setup call creates the diesel record table. I guess you could revert it back to explicit sql if it's hard to turn it into async

Reply:
I don't believe this explicit call to setup is necessary, without it all the tests still pass properly. Its likely handled as a part of the run_pending migrations flow. I'll remove the commented out line though